### ISSUE TITLE
LR scheduler from config file doesn't work

### ISSUE THREAD

**Issue by siemdejong** [siemdejong/sclicom#28]
**Describe the bug**
The learning rate is not changed when using a config file to choose the learning rate scheduler.

**Expected behavior**
Change the learning rate according to the chosen learning rate scheduler.

**Screenshots**
![image](https://user-images.githubusercontent.com/28396796/227175055-84b0e488-e202-4f93-8016-e6d509bde9cb.png)

**Desktop (please complete the following information):**
 - OS: [e.g. Linux, Windows, macOS]
 - DPAT version 4.1.0

**Additional context**
Using the varmil.yaml config file with
```yaml
  model:
    scheduler:
      class_path: torch.optim.lr_scheduler.CosineAnnealingLR
      init_args:
        T_max: ${trainer.max_epochs}
```

**Comment by siemdejong** [comment]
Fixed in https://github.com/siemdejong/dpat/commit/d2b62e1631c88db8914497a55baf8a45d04a1f3a
